### ISSUE TITLE
Not downloading whole image

### ISSUE THREAD

**Issue by siobhanjackson** [lvandernoll/pokecardmaker.net#356]
1. Hey I’ve tried to use the card make a few times but when I try to download it either has no image, only text, image and text but no white text behind but it won’t download fully, not sure if I’m doing something wrong. I’ve tried on my phone and iPad 
...
![C5F1C98B-98FC-474C-83D7-677A5A3DAB3D](https://user-images.githubusercontent.com/2835549/214701879-e27b6217-b036-478b-8a65-537880f1298d.png)
![7D5F21BD-7C36-49B5-A686-927772C7E7B3](https://user-images.githubusercontent.com/2835549/214701886-2f61e2bc-f2e9-4321-bf12-57ead6ef1388.png)

**Screenshots**
...

**Comment by ZerglingH** [comment]
Same thing for me... I download the card, but with no images... :(

**Comment by lvandernoll** [comment]
Is this a new issue? @ZerglingH I'm pretty sure I've seen you around the GitHub discussions page before, so did it work before for you?

**Comment by ZerglingH** [comment]
No, it's about 2 month and i can't download the card with the image so, the only thing that i can do is to take a screenshot of my card.

**Comment by lvandernoll** [comment]
Interesting to see that only the card images don't work for you, and the set icon works fine. Could you try whether it works in an incognito tab, or a different browser?

**Comment by ZerglingH** [comment]
It's working in a incognito tab: 
![Bulbasaur (6)](https://user-images.githubusercontent.com/118122812/215284635-e6b8a3b0-87bd-40bd-8986-347710236889.png)

**Comment by lvandernoll** [comment]
That means it's probably an issue with the extensions you're using in your browser. Maybe some adblocker?

**Comment by ZerglingH** [comment]
I see the images with the card on the website, but when I download the card, all the images disappears except the card base and other things. So, I have to take a screenshot of my card to download it instead of download it normaly.

**Comment by lvandernoll** [comment]
But it does work in an incognito tab you said, right? In that case it should be fixed when disabling some extensions. Could you check this? You could start by disabling any adblocker and popup blocker for the site. There's no ads or popups anyways

**Comment by kintaro1981** [comment]
can't this be related to [this](https://github.com/lvandernoll/pokecardmaker.net/issues/349)?

**Comment by VOtolotoo2** [comment]
I have the same problem but screenshot works

**Comment by ZerglingH** [comment]
@VOtolotoo2 I'm not complaining too with a screenshot.

@lvandernoll It's now working without an incognito tab (i don't know why it wasn't working). I can now create my card and download them in peace. :D
